### DISCUSSION TITLE
Heatblock and Thermistor incompatibly

### DISCUSSION THREAD

**Discussion by LimesKey** [bassamanator/everything-sovol-sv06#21] [General]
I bought 2 of the most popular CR10 Heatblocks off [AliExpress ](https://www.aliexpress.com/item/1005004509191950.html?spm=a2g0o.order_list.order_list_main.114.7a571802eGYN3J)to replace my broken heat block. The issue I'm running into is that the thermistor cartridge is too large for the heat block, I also bought 4 different brands of thermistor cartridges, which are all too large for it. Can you point me in the direction of a CR10 heat block which is bigger; or then a smaller thermistor cartridge? I'm tempted to just drill the heat blocks to make space for the thermistor.

Heat blocks I bought, [1](https://www.aliexpress.com/item/1005004509417894.html?spm=a2g0o.order_list.order_list_main.108.7a571802eGYN3J), [2](https://www.aliexpress.com/item/1005004509191950.html?spm=a2g0o.order_list.order_list_main.114.7a571802eGYN3J)
Thermistor Cartridges I bought, [1](https://www.aliexpress.com/item/32968569888.html?spm=a2g0o.order_list.order_list_main.102.7a571802eGYN3J), [2](https://www.aliexpress.com/item/32817471679.html?spm=a2g0o.order_list.order_list_main.96.7a571802eGYN3J), [3](https://www.aliexpress.com/item/32827784267.html?spm=a2g0o.order_list.order_list_main.84.7a571802eGYN3J), [4](https://www.aliexpress.com/item/32827784267.html?spm=a2g0o.order_list.order_list_main.52.7a571802eGYN3J)

Here are some photos, 
![image](https://user-images.githubusercontent.com/85136735/226703857-c2c51f22-61a8-4b75-97a1-9c5ede23eead.png)
![image](https://user-images.githubusercontent.com/85136735/226703869-6dd69e96-cb8d-4477-8220-01124f5e3419.png)

**Comment by LimesKey** [comment]
All the thermistors I have are a 3mm diameter, while the hole is 2mm

**Comment by LimesKey** [comment]
After some more digging, the CR10 heat block that Sovol uses only supports [Glass Bead Thermistors ](https://www.aliexpress.com/item/32514751406.html?spm=a2g0o.productlist.main.13.4fc51vs51vs5FC&algo_pvid=4a9f6fcc-9db5-4db7-91e2-68112b03071e&aem_p4p_detail=202303220250351111396485568820005062336&algo_exp_id=4a9f6fcc-9db5-4db7-91e2-68112b03071e-6&pdp_ext_f=%7B%22sku_id%22%3A%2265732281353%22%7D&pdp_npi=3%40dis%21CAD%211.63%211.54%21%21%21%21%21%402102176616794786350847201d0767%2165732281353%21sea%21CA%214666093930&curPageLogUid=ShRbt8qZkw2q&ad_pvid=202303220250351111396485568820005062336_7&ad_pvid=202303220250351111396485568820005062336_7)that are 2mm in diameter. To be able to use a [cartridge thermistor ](https://www.aliexpress.com/item/32827784267.html?spm=a2g0o.productlist.main.21.4fc51vs51vs5FC&algo_pvid=4a9f6fcc-9db5-4db7-91e2-68112b03071e&algo_exp_id=4a9f6fcc-9db5-4db7-91e2-68112b03071e-10&pdp_ext_f=%7B%22sku_id%22%3A%2212000024684828215%22%7D&pdp_npi=3%40dis%21CAD%212.08%211.98%21%21%21%21%21%402102176616794786350847201d0767%2112000024684828215%21sea%21CA%214666093930&curPageLogUid=jxyDNDvUUGZ6)that I have (and like better) you'll have to drill it out with a 1/8 drill bit and drill very slowly and carfully not to drill into the threading of the M6 hotend nozzle.

**Comment by BeauSlim** [comment]
I haven't tried one, but Mellow has a couple (copper and aluminum) that have spots for both glass bead and cartridge thermistors:
https://www.aliexpress.com/item/33051274625.html

**Comment by LimesKey** [comment]
> I haven't tried one, but Mellow has a couple (copper and aluminum) that have spots for both glass bead and cartridge thermistors: https://www.aliexpress.com/item/33051274625.html

That is great, 1 small issue, the stock screws that come with the heatblock from SV06 are not long enough and do not fit in the mounting holes in the heatblock I have, and with the Mellow's heatblock. I tried drilling it but was unsuccessful and I will have to get longer screws

**Comment by LimesKey** [comment]
So then I drilled it with a 3/16 bit to widen the hole and to eventually put a washer on it but I'll still need a longer screw. Maybe next time I can drill slower and leave a small 1mm platform where the screw can sit on and hold onto the heatblock to mount it. 

The heatblock mounting mechanism on the ones from AliExpress are different from the stock ones, with the heatblock from AliExpress I need longer screws. The current screws that were stock are 2.2mm in diameter and the entire height of the screw is 12.2mm in height, I think I'll need 16mm in height.

![image](https://user-images.githubusercontent.com/85136735/227401415-2cac3d65-a244-4561-8352-13764eaccb01.png)
I have only 1 extra heatblock untouched in case I messed up the drilling and need to restart, but if you compare the stock and one from AliExpress heatblocks side by side you'll see.
![image](https://user-images.githubusercontent.com/85136735/227402559-fc223e4d-6549-4e5c-8209-64c1761ecb16.png)
The one from AliExpress is the brand new one which has a circular indent in it while the stock one is just a flat cylinder.

**Reply by flashgordon1427** [reply]
The link included for the heatblock option from AliExpress has multiple versions of the heatblock. Option B has countersunk holes that align more with the stock heatblock, I wonder if that would solve the screw length issue

**Reply by albertoabrao** [reply]
I bought the SV06 after getting really frustrated with my Ender 3 S1.

As a result, I had to try both the thermistor and the heating element that I had on hand:

1) The thermistor for the Ender 3 S1 works, no crimping needed. For [this](https://www.aliexpress.com/item/4000235433543.html?spm=a2g0o.order_list.order_list_main.11.6d131802HEuJWu) heatblock, it was plug-and-play. 

If you have a standard CR10 heatblock, you can heat the thermistor with a heatgun, then pull the cable. I did it successfully by going up to 300C and a good, confident tug on the cable to take it out of its enclosure.  I strongly recommend using one that lets you use it as-is, makes it easier - other than the longer, countersunk head screws it requires.

The heatblock I linked above is plug-and-play for the Ender 3 S1 thermistor. I used a little of boron nitride paste just to hold it in place, routed the cable around the screw, then tightened it gently. I was afraid the insulation would let go, but it's been like that for a few months now, and I print every day, PETG always (240C). So far, so good.

2) The heating element for the Ender 3 S1 also works, but you'll have to crimp the connector as explained on the main page of bassamanator's repo. The beauty is that the wires are just the right length after cutting the old connector out, stripping the ends, and crimping the new one. I was afraid it'd be awful, but other than making sure you have a good connection, the crimp job is made easier by the wires being stiff and short. just remember to keep the insulation close to the heating element while crimping, as it "dances" on the wire. If that doesn't make sense to you right now, it will then :)

If it helps anyone, here's the [link](https://www.aliexpress.com/item/1005005271289566.html?spm=a2g0o.order_list.order_list_main.5.6d131802HEuJWu) I used to buy my spares last time.

Oh, and one more thing: easy on the torque!! It's easy to bend the throat... or the screws that attach the heatblock to the extruder. I personally do it before assembly, all parts cold. Torque to around 1.6Nm, if anyone cares.
